web/client/selectors/dynamiclegend.js

```diff
@@ -0,0 +1,14 @@
+const isLayer = node => node.nodeType === "layers";

```

Comment:
In https://github.com/allyoucanmap/MapStore2/commit/a11b6736dd0cc5175977ec75a212d3c7d205d309 this file has been renamed to DynamicLegendUtils because it does not contain selectors but utils. Some of the utils has been changed to support the new hiding logic.

To better isolate the plugin all files have been moved under the `plugins/DynamicLegend` folder:

```
plugins/DynamicLegend/
    |__assets/
    |__components/
    |__utils/
    |__index.js
```

Note `DynamicLegend` is uppercase as the name file so the import resolver will look for the index.js file inside